dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java

```diff
@@ -300,6 +300,13 @@ private List<ContainerRaw> populateContainers() throws DotDataException, DotSecu
                     final Contentlet contentlet = transformer.hydrate().get(0);
                     this.addContentletPageReferenceCount(contentlet);
 
+                    // Skip content with invalid date configuration in LIVE mode
+                    if (mode.showLive && hasInvalidDateConfiguration(contentlet)) {
+                        Logger.debug(this, ()-> "Skipping contentlet " + contentlet.getIdentifier()
```

Comment:
I think it makes more sense to make this a warning, saying that we have ignored or skipped a piece of content that has an invalid config

Reply:
perhaps the warning must be displayed for every piece of content that has an invalid configuration regardless the mode And skip it only for live mode

Reply:
also I thik this exclusion logic is the first thing that needs to happen here 
e.g before this.addContentletPageReferenceCount(contentlet);